main/windows/index.ts

```diff
@@ -36,41 +36,48 @@ let dawn: Dawn
 let mouseTimeout: NodeJS.Timeout
 let glide = false
 
-const enableHMR = process.env.NODE_ENV === 'development' && process.env.HMR === 'true'
-const hideFrame = () => tray.hide()
-const showFrame = () => tray.show()
-
-const separatorMenuItem = {
-  label: 'Frame',
-  click: () => {},
-  type: 'separator'
-}
+const enableHMR = isDev && process.env.HMR === 'true'
+const contextMenu = (displaySummonShortcut: boolean = store('main.shortcuts.altSlash')) => {
```

Comment:
functionally I think this makes sense. I think it might read a little more clearly if all of the context menu stuff was in its own module which exposed a clear API, either two functions like `createShowContextMenu()` and `createHideContextMenu()` or `createContextMenu('show')` or something like that

Reply:
agreed, would make for a smaller windows module too - I think it's too big currently